src/components/Engines/Engines.test.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+jest.mock('./img/node.png', () => '');
```

Comment:
We should not mock this due to we uses `identity-obj-proxy` to do it out of the box, but If I remove the mock the test fails, thus, the configuration seems does not work fine. I'll address this in a new ticket.
https://github.com/keyz/identity-obj-proxy
See example here.
https://github.com/verdaccio/ui/blob/4.x-master/jest.config.js#L16

Reply:
https://github.com/verdaccio/ui/issues/93

Reply:
Yes. You are right. I'll investigate it. Nice catch 👍